app/services/reports/sp_active_users_over_period_of_performance_report.rb

```diff
@@ -0,0 +1,14 @@
+require 'login_gov/hostdata'
+
+module Reports
+  class SpActiveUsersOverPeriodOfPerformanceReport < BaseReport
```

Comment:
What if we called this the "Since april 1 report" or something, or included that in the name?

Reply:
Yeah.  Wasn't happy with the name from the get go.

Reply:
Tho that is the name coined by @sdannemann

Reply:
It was just an idea, if Silke is the main consumer and this helps with stuff, that's fine

Reply:
I don't feel that strongly about the name, keeping it is fine